go.mod

```diff
@@ -160,10 +160,17 @@ require (
 
 require github.com/google/gnostic v0.5.7-v3refs
 
-require github.com/kr/fs v0.1.0 // indirect
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.0.0 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.0.0 // indirect
+	github.com/kr/fs v0.1.0 // indirect
+	github.com/pkg/browser v0.0.0-20210115035449-ce105d075bb4 // indirect
```

Comment:
We adding a few things to the go.mod file. Could explain why we need all of these? I don't want to bloat this file too much as it's rancher/rancher's main go.mod file